docs/current/sdk/nodejs/reference/modules.md

```diff
@@ -11,4 +11,5 @@ displayed_sidebar: "current"
 ## Modules
 
 - [api/client.gen](modules/api_client_gen.md)
+- [common/errors](modules/common_errors.md)
```

Comment:
This is the issue I mentioned earlier - this file is auto-generated when the website is tested locally and pollutes the commit

Reply:
I remember.

I am going to drop this change & run `git update-index --skip-worktree docs/current/sdk/nodejs/reference/` locally:

<img width="984" alt="image" src="https://user-images.githubusercontent.com/3342/206480396-6a5fb745-c91a-4e53-9ebd-867aeeabd704.png">

I am think that we should gitignore that entire path, and stop tracking generated files in git. Is there a reason why we are not doing that already?

Reply:
<img width="773" alt="image" src="https://user-images.githubusercontent.com/3342/206488005-c2476123-3fce-47ae-80f8-6caad52bc2cf.png">

Reply:
> I am think that we should gitignore that entire path, and stop tracking generated files in git. Is there a reason why we are not doing that already?

The sidebar includes a hard link to reference/modules.md so it was not possible to gitignore it at the very beginning, as that would have excluded it from the repository and Docusaurus would have thrown a broken link error. However, now that the file is part of the repository, we could indeed add it to .gitignore so future changes to it are not tracked.